src/authenticationinapp/authenticationinapplistener.cpp

```diff
@@ -149,7 +149,7 @@ void AuthenticationInAppListener::checkAccount(const QString& emailAddress) {
 
   connect(request, &NetworkRequest::requestCompleted, this,
           [this](const QByteArray& data) {
-            logger.debug() << "Account status checked" << data;
+            logger.debug() << "Account status checked";
```

Comment:
can we use the `sensitive()` method instead? These messages are still useful for debugging

Reply:
Okay, I'll convert these to sensitive instead.